modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10594,6 +10594,7 @@ object-action-trigger[onAfterUpdate]=On After Update
 object-configuration-name=Object
 object-definition-data=Object Data
 object-definition-id=Object ID
+object-definition-storage-type-tooltip=Select if the data will be stored in Liferay (default) or in an external system only. Limitations are applied for object definitions with external data bases with external services.
```

Comment:
Hey @alinedoleron , can you change the hint message to end with: **Limitations are applied for object definitions with external services.** ?